cmd/geth/misccmd.go

```diff
@@ -103,7 +103,7 @@ func version(ctx *cli.Context) error {
 	}
 	fmt.Println("Architecture:", runtime.GOARCH)
 	fmt.Println("Protocol Versions:", eth.ProtocolVersions)
-	fmt.Println("Network Id:", ctx.GlobalInt(utils.NetworkIdFlag.Name))
+	fmt.Println("Network Id:", eth.DefaultConfig.NetworkId)
```

Comment:
Yeah, when I touched this line I though the same but didn't want to make these kind of changes within the scope of this PR. Was afraid I may end up changing more similar things which could get ugly.